### ISSUE TITLE
Migration Plan wizard changes for warm migration

### ISSUE THREAD

**Issue by vconzola** [ManageIQ/manageiq-v2v#1004]
Mockups: 
[Mockups for Warm Migration - Plan Wizard Changes v0.1.pdf](https://github.com/ManageIQ/manageiq-v2v/files/3407828/Mockups.for.Warm.Migration.-.Plan.Wizard.Changes.v0.1.pdf)

**Comment by mzazrivec** [comment]
@vconzola Vince, I have a question about info tooltip icon, which is shown in the mockups above next to `Migration Type` label. What text is the tooltip supposed to show?

Thanks.

![migrations](https://user-images.githubusercontent.com/6648365/62558951-500c1200-b87a-11e9-83fe-c216bf0f5559.png)

**Comment by mturley** [comment]
@mzazrivec did you ever get an answer from @vconzola about this info tooltip icon text?

**Comment by mzazrivec** [comment]
@mturley Yep, I have the tooptip icon text in the mockups from Vince, the only thing I'm lacking right now is the validation API for it.
